www/nginx/src/opnsense/service/templates/OPNsense/Nginx/newsyslog.conf

```diff
@@ -0,0 +1,5 @@
+# logfilename                   [owner:group]   mode    count size      when    flags   [/pid_file]               [sig_num]
+{% if helpers.exists('OPNsense.Nginx') %}
+/var/log/nginx/*access.log   www:www     644     14       *      @T00     GZB      /var/run/nginx.pid       30
+/var/log/nginx/*error.log   www:www     644     14       *      @T00     GZB      /var/run/nginx.pid       30nginx-
```

Comment:
the pattern looks ok but logs should not be world readable, so 640 is probably better. Can you explain me the nginx- suffix?

Reply:
About the file permissions, OK!

I've done this way, because in one of my installations we have a lot of log different filenames and in a simplistic manner (wildcard G flag), all logs are being rotated. To make it better, I think we will need some Log options in WebUI and better integration... TODO to a next version.